README.md

```diff
@@ -12,12 +12,19 @@ and checks code against the coding standards used in CakePHP.
 PSR2 standard. If you want to check against the historical CakePHP coding
 standard use any of the `1.x` releases.
 
+## Which version should I use?
+| CakePHP  | Sniffer |
+| -------- | ------- |
+| 2.x  | 1.0.4  |
+| 3.x  | latest 3.x tag  |
+| 4.x  | latest 4.0.0-beta  |
```

Comment:
```
| 4.x  | latest 3.x  |
```

Reply:
There is no `4.x` branch in this repo, it's `dev` :slightly_smiling_face:.

Reply:
They aren't branches, just releases.

Reply:
Ah, okey dokey.

Reply:
Still, semantically >= is more helpful.